docs/wallet-extension/configure-metamask.md

```diff
@@ -0,0 +1,12 @@
+# Configuring MetaMask
```

Comment:
I thought we had a nice script which does this on the guessing game page? Can this be added to the configure-metamask page?

Reply:
It was being repeated on multiple pages, so I pulled it out and made the steps simpler.